updater/lib/dependabot/sentry.rb

```diff
@@ -1,15 +1,28 @@
 # frozen_string_literal: true
 
-require "raven"
-
 # ExceptionSanitizer filters potential secrets/PII from exception payloads
-class ExceptionSanitizer < Raven::Processor
+class ExceptionSanitizer
   REPO = %r{[\w.\-]+/([\w.\-]+)}
   PATTERNS = {
     auth_token: /(?:authorization|bearer):? (\w+)/i,
     repo: %r{api\.github\.com/repos/#{REPO}|github\.com/#{REPO}}
   }.freeze
 
+  def process_sentry(event, hint)
+    # TODO migrate these payloads from the old "data" to the new "event"/"hint"
```

Comment:
This should be quick if I can get into the debugger to see the payload, but for some reason when I drop a `debugger` statement here, isn't firing during unit tests... Maybe they've mocked the actual Sentry call so the caller of this (`before_send()`) never fires?